src/System.IO/ref/System.IO.cs

```diff
@@ -164,6 +164,11 @@ public partial class StreamReader : System.IO.TextReader
         public StreamReader(System.IO.Stream stream, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks) { }
         public StreamReader(System.IO.Stream stream, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks, int bufferSize) { }
         public StreamReader(System.IO.Stream stream, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks, int bufferSize, bool leaveOpen) { }
+        public StreamReader(string path) { }
+        public StreamReader(string path, bool detectEncodingFromByteOrderMarks) { }
+        public StreamReader(string path, System.Text.Encoding encoding) { }
+        public StreamReader(string path, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks) { }
+        public StreamReader(string path, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks, int bufferSize) { }
```

Comment:
I just realized... this is using the coreclr StreamReader, right?  Which means it's using the coreclr implementation of FileStream rather than the one in corefx?  I'm concerned there may be some resulting issues.  For example, from a compat standpoint, StreamReader exposes BaseStream, which will be a different FileStream than the one exposed publicly.  And has all of the work been done to make FileStream in coreclr work with long paths as was done in corefx?  What about all of the various changes we've done in UnixFileStream in corefx to support various unix-specific thins? Etc.